scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/SimulationFacade.java

```diff
@@ -183,6 +184,9 @@ public void removeAndInsertActivitiesFromSimulation(
       final Collection<SchedulingActivityDirective> activitiesToRemove,
       final Collection<SchedulingActivityDirective> activitiesToAdd) throws SimulationException
   {
+    logger.info("Removing("+activitiesToRemove.size()+")/Adding("+activitiesToAdd.size()+") activities from simulation");
+    activitiesToRemove.stream().forEach(remove -> logger.info("Removing act starting at " + remove.startOffset()));
+    activitiesToAdd.stream().forEach(adding -> logger.info("Adding act starting at " + adding.startOffset()));
```

Comment:
Minor, can ignore: The number of activities being swapped out and when they are is only needed during debugging
```suggestion
    logger.debug("Removing("+activitiesToRemove.size()+")/Adding("+activitiesToAdd.size()+") activities from simulation");
    activitiesToRemove.stream().forEach(remove -> logger.debug("Removing act starting at " + remove.startOffset()));
    activitiesToAdd.stream().forEach(adding -> logger.debug("Adding act starting at " + adding.startOffset()));
```